sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -147,22 +163,33 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
           className="not-prose"
         >
           <Link href="/[username]" as={`/${username}`} passHref>
-            <a>
-              <Text size="action">{siteName}</Text>
-              <Text
-                size="action"
-                css={{
-                  display: 'flex',
-                  gap: '$2',
-                  color: '$gray9',
-                  mt: '$1',
-                }}
-              >
-                {format(story.createdAt, 'MMM dd')}
-                <span>•</span>
-                <span>{storyReadingTime?.text}</span>
-              </Text>
-            </a>
+            <Flex as="a" align="center" gap="3">
+              <Box
+                as="img"
+                src={
+                  settingsFile?.siteLogo
+                    ? settingsFile.siteLogo
+                    : sigleConfig.boringAvatarUrl
+                }
+                css={{ width: 48, height: 48, br: '$1' }}
```

Comment:
should probably set the fit properties here too otherwise image might look different